examples/mongocxx/mongodb.com/documentation_examples.cpp

```diff
@@ -1180,6 +1193,85 @@ void delete_examples(mongocxx::database db) {
     }
 }
 
+static void snapshot_examples(mongocxx::client& client) {
+    // Write concern "majority" needed to avoid the following error below:
+    //
+    // Unable to read from a snapshot due to pending collection catalog changes;
+    // please retry the operation.
+    //
+    // See also: https://jira.mongodb.org/browse/SERVER-41532
+
+    // Start Example 59
+    using namespace mongocxx::v_noabi;
```

Comment:
```suggestion
    using namespace mongocxx;
```

`v_noabi` is an [inline namespace](https://github.com/mongodb/mongo-cxx-driver/blob/e6aeda6575ee0f7338019edf812055370f7f9c8c/src/mongocxx/config/config.hpp.in#L15).